maplibre_gl_example/lib/line.dart

```diff
@@ -66,6 +69,16 @@ class LineBodyState extends State<LineBody> {
     );
   }
 
+  _onFeatureHover(Annotation annotation, Point<double> point, LatLng latLng,
+      HoverEventType eventType) async {
+    if (annotation is! Line) return;
+    if (eventType == HoverEventType.enter) {
+      controller!.updateLine(annotation, const LineOptions(lineWidth: 16));
+    } else if (eventType == HoverEventType.leave) {
+      controller!.updateLine(annotation, const LineOptions(lineWidth: 14));
+    }
+  }
```

Comment:
This changes makes the example clearer when users hover over the features
```suggestion
  _onFeatureHover(Annotation annotation, Point<double> point, LatLng latLng,
      HoverEventType eventType) async {
    if (annotation is! Line) return;
    if (eventType == HoverEventType.enter) {
      controller!.updateLine(
          annotation,
          const LineOptions(
            lineWidth: 16,
            lineColor: "#8B0000",
          ));
    } else if (eventType == HoverEventType.leave) {
      controller!.updateLine(
          annotation,
          const LineOptions(
            lineWidth: 14,
            lineColor: "#ff0000",
          ));
    }
  }
```